index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Calculator Benchmark</title>
+  <meta charset="utf-8">
+  <meta name="description" content="calculator">
+  <meta name="author" content="Keith Oppel">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+  <div class="calculator-container">
+    <div class="calculator-display">
+      <span class="display">0</span>
+    </div>
+    <div class="calculator-row">
+      <div class="calculator-button"><button value="AC">AC</button></div>
+      <div class="calculator-button"><button value="±">+/-</button></div>
```

Comment:
Make sure you press enter after the div is created. It can be hard to be tags close together.